BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -19,7 +19,7 @@
 @protocol BOXURLSessionManagerDelegate;
 @protocol UniqueSDKUser;
 
-typedef void (^ServerAuthFetchTokenBlock)(NSString *, NSDictionary *, void (^)(NSString *, NSDate *, NSError *));
+typedef void (^ServerAuthFetchTokenBlock)(NSString * _Nonnull, NSDictionary * _Nullable, void (^ _Nonnull)(NSString * _Nullable, NSDate * _Nullable, NSError * _Nullable));
```

Comment:
We could even write: `typedef void (^ServerAuthFetchTokenBlock)(NSString * _Nonnull uniqueID, NSDictionary * _Nullable userInfo, void (^ _Nonnull completion)(NSString * _Nullable, NSDate * _Nullable, NSError * _Nullable));` and users will get the default parameter names autocompleted.

Reply:
Right now, I find this declaration inscrutable.

Reply:
That's true of the completion block, too, and we could add names there, but I don't think users will generally see them.